src/Common/Tokenizer/ChineseTokenizer.cpp

```diff
@@ -0,0 +1,32 @@
+#include "ChineseTokenizer.h"
+
+#include <cassert>
+
+#include <base/defines.h>
+#include <cppjieba/Jieba.hpp>
+
+namespace DB
+{
+
+ChineseTokenizer & ChineseTokenizer::instance()
+{
+    static ChineseTokenizer tokenizer{};
+    return tokenizer;
+}
+
+std::vector<std::string> ChineseTokenizer::tokenize(const std::string & str)
+{
+#ifdef DEBUG_OR_SANITIZER_BUILD
```

Comment:
The singleton code is quite simple, let's remove l. 19-21 ... it is too obvious that the assert can never be hit.